x/stakeibc/client/cli/tx_register_community_pool_rebate.go

```diff
@@ -0,0 +1,54 @@
+package cli
+
+import (
+	"errors"
+	"fmt"
+
+	sdkmath "cosmossdk.io/math"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+
+	"github.com/Stride-Labs/stride/v19/x/stakeibc/types"
+)
+
+func CmdRegisterCommunityPoolRebate() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "register-rebate [chain-id] [rebate-percentage] [liquid-staked-amount]",
+		Short: "Registers a community pool rebate",
+		Args:  cobra.ExactArgs(3),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			chainId := args[0]
+			rebatePercentage, err := sdk.NewDecFromStr(args[1])
+			if err != nil {
+				return fmt.Errorf("unable to parse rebate percentage: %s", err.Error())
+			}
+			liquidStakeAmount, ok := sdkmath.NewIntFromString(args[2])
```

Comment:
should we just make this a uint to enforce it's gte 0?

Reply:
I think we should leave it as an sdk.Int, otherwise, we'll have to cast it every time it's used. It's easy enough to enforce gt 0 in validate basic imo

Reply:
sg